src/config/layout.rs

```diff
@@ -20,22 +20,22 @@ pub enum Layout {
     LeftWiderRightStack,
 }
 
-pub const LAYOUTS: &[Layout] = &[
-    Layout::MainAndVertStack,
-    Layout::MainAndHorizontalStack,
-    Layout::MainAndDeck,
-    Layout::GridHorizontal,
-    Layout::EvenHorizontal,
-    Layout::EvenVertical,
-    Layout::Fibonacci,
-    Layout::LeftMain,
-    Layout::CenterMain,
-    Layout::CenterMainBalanced,
-    Layout::CenterMainFluid,
-    Layout::Monocle,
-    Layout::RightWiderLeftStack,
-    Layout::LeftWiderRightStack,
-];
+//pub const LAYOUTS: &[Layout] = &[
+//    Layout::MainAndVertStack,
+//    Layout::MainAndHorizontalStack,
+//    Layout::MainAndDeck,
+//    Layout::GridHorizontal,
+//    Layout::EvenHorizontal,
+//    Layout::EvenVertical,
+//    Layout::Fibonacci,
+//    Layout::LeftMain,
+//    Layout::CenterMain,
+//    Layout::CenterMainBalanced,
+//    Layout::CenterMainFluid,
+//    Layout::Monocle,
+//    Layout::RightWiderLeftStack,
+//    Layout::LeftWiderRightStack,
+//];
```

Comment:
Possibly remove the commented code fully if we're sure that we're not going to use it again.